esphome/components/i2c/i2c.cpp

```diff
@@ -178,28 +178,47 @@ bool I2CComponent::write_byte_16(uint8_t address, uint8_t a_register, uint16_t d
 }
 
 void I2CDevice::set_i2c_address(uint8_t address) { this->address_ = address; }
+void I2CDevice::set_i2c_multiplexer(I2CMultiplexer *multiplexer, uint8_t channel) {
+  ESP_LOGVV(TAG, "    Setting Multiplexer %p for channel %d", multiplexer, channel);
+  this->multiplexer_ = multiplexer;
+  this->channel_ = channel;
+}
+void I2CDevice::check_multiplexer_() {
+  if (this->multiplexer_) {
```

Comment:
To keep consistency with other code that checks for `nullptr`

```suggestion
  if (this->multiplexer_ != nullptr) {
```